managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/PositionImpl.java

```diff
@@ -129,6 +129,20 @@ public boolean equals(Object obj) {
         return false;
     }
 
+    public static PositionImpl convertStringToPosition(String positionString) {
+        if (positionString == null) {
+            throw new NullPointerException();
+        } else {
+            String[] strings = positionString.split(":");
+            if (strings.length != 2) {
+                throw new IndexOutOfBoundsException();
```

Comment:
can you add a message ? like "invalid position "+positionString
it will ease debugging issues in the future

Reply:
sure, it is a good idea!